mock/data/law.ts

```diff
@@ -0,0 +1,32 @@
+import { LawOfferDefinitionDTO, OffersLawOffer, LawOffer } from '@app/core/api';
+
+export const lawOffer = (body: LawOfferDefinitionDTO): LawOffer => {
+  return {
+    ...body,
+    id: 1,
+    userFirstName: 'example',
+    modifiedDate: new Date().toISOString(),
+    type: 'LAW',
```

Comment:
da się to wziąć z `LawOffer.TypeEnum.Law`?

Reply:
Poprawione